src/app/OperationalSessionSetup.h

```diff
@@ -202,6 +202,8 @@ class DLL_EXPORT OperationalSessionSetup : public SessionDelegate,
 
     bool IsConnecting() const { return mState == State::Connecting; }
 
+    bool IsSessionDesignatedLookup() const { return mPerformingLookupOnConnectedSession; }
```

Comment:
```suggestion
    bool IsJustAddressUpdate() const { return mPerformingAddressUpdate; }
```
might be better naming...  At this point this thing is not really tied to a "session".  It's just resolving an updated address for a given peer.

Reply:
Went with `IsForAddressUpdate`